Add Windows

jakirkham opened the pull request:
As of `glog` 0.3.5, `cmake` builds are supported. Given this, try and build `glog` 0.3.5 on Windows.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham committed:
Add selectors for Unix only dependencies

Go ahead and add Unix selectors for the Unix only dependencies in
preparation of adding Windows builds.

jakirkham committed:
Drop Windows skip from recipe

jakirkham committed:
Add VC features for Windows

jakirkham committed:
Add Windows requirements

jakirkham committed:
Add Windows build script using CMake

jakirkham committed:
Skip Python 3.5 on Windows

As Python 3.5 and 3.6 use the same VC version, there is no need to build
for both. So simply chose the most recent of the two Pythons.

jakirkham committed:
MNT: Re-rendered with conda-smithy 2.3.2

Enable Windows builds of this package now that they are feasible.

jakirkham committed:
Require Python 3.6 to build on Windows

Appears that `glog` now requires C++11 support. As a consequence, it is
simply not possible to build using Visual C++ 2008, which does not have
C++11 support. Give that Python 3.5 and 3.6 both use the same VC
runtime, simply require Python 3.6 and up to build `glog` on Windows.

jakirkham committed:
MNT: Re-rendered with conda-smithy 2.3.2

Disable Visual C++ 2008 based builds on Windows as glog requires C++11
support, which that older runtime does not have.

jakirkham commented:
Appears the 64-bit Windows build is failing, but the 32-bit Windows build works fine. Have raised an issue upstream ( https://github.com/google/glog/issues/202 ).

jakirkham commented:
This PR is pretty out-of-date. So someone would probably want to start a new PR, but I would guess it is much easier to do today if it is of interest.
